tests/functional/test_download.py

```diff
@@ -849,3 +849,19 @@ def test_download_http_url_bad_hash(
     assert len(requests) == 1
     assert requests[0]['PATH_INFO'] == '/simple-1.0.tar.gz'
     assert requests[0]['HTTP_ACCEPT_ENCODING'] == 'identity'
+
+
+def test_download_editable(script, data, tmpdir):
+    """
+    Test 'pip download' of editables in requirement file.
+    """
+    editable_path = data.src / 'simplewheel-1.0'
+    requirements_path = tmpdir / "requirements.txt"
+    requirements_path.write_text("-e " + str(editable_path.resolve()) + "\n")
```

Comment:
> ERROR: R:Temppytest-of-VssAdministratorpytest-1popen-gw1test_download_editable0datasrcsimplewheel-1.0 is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with svn+, git+, hg+, or bzr+).

@pfmoore @uranusjr would you have any idea why `os.path.realpath` that's running behind this call is messing up the contents of this file?

Reply:
This is happening only one the Windows runners, and "it works on my machine". 😅